bouncer/shared/new_swap.ts

```diff
@@ -17,14 +17,19 @@ export async function newSwap(
   brokerCommissionBps = defaultCommissionBps,
 ): Promise<void> {
   const destinationAddress =
-    destAsset === 'DOT' ? decodeDotAddressForContract(destAddress) : destAddress;
+    destAsset === 'Dot' ? decodeDotAddressForContract(destAddress) : destAddress;
   const brokerUrl = process.env.BROKER_ENDPOINT || 'http://127.0.0.1:10997';
 
   await broker.requestSwapDepositAddress(
     {
-      // Temporal workaround for ARB assets
-      srcAsset: sourceAsset.replace('ARB', '') as Asset,
-      destAsset: destAsset.replace('ARB', '') as Asset,
+      // TODO: Temporal workaround: To remove once SDK supports Arbitrum
+      srcAsset: (assetConstants[sourceAsset]
+        ? stateChainAssetFromAsset(sourceAsset)
+        : sourceAsset.toUpperCase().replace('ARB', '')) as SCAsset,
```

Comment:
could support the arbitrum assets in the `stateChainAssetFromAsset` method now

Reply:
Right, that is a nicer workaround. Will do!